extensions/wikia/Listusers/SpecialListusers_helper.php

```diff
@@ -466,12 +456,12 @@ public function updateUserGroups( $user, $addgroup = array(), $removegroup = arr
 			}
 
 			$dbw->replace(
-				$this->mTable,
-				array( 'wiki_id', 'user_id', 'user_name' ),
+				self::TABLE,
+				array( 'wiki_id', 'user_id' ),
 				array(
 					"wiki_id"        => $this->mCityId,
 					"user_id"        => $user_id,
-					"user_name"  	 => $user->getName(), # TODO: SUS-3205
+					"user_name"  	 => '', # TODO: SUS-3204 - insert either user ID or user name
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this "TODO" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)

[project-id]: wikia:app
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)

[project-id]: wikia:app